capstone-ui/src/components/App/App.jsx

```diff
@@ -112,33 +129,72 @@ export default function App() {
         .then(function (response) {
           setUserInfo(response.data.userInfo);
           setUserMatches([]);
-          setTimeout(getInterestsFromUser, 500)
-          getMatchesForUser(matchLimit + matchOffset, 0)
-          setTimeout(setFetchingFalse, 1500)
+          setTimeout(getInterestsFromUser, 500);
+          getMatchesForUser(matchLimit + matchOffset, 0);
+          setTimeout(setFetchingFalse, 1500);
         })
         .catch(function (err) {
           console.log(err);
         })
     }
   }
 
-  async function getSpotifyUser(access_token) {
-    setIsFetching(true);
+  async function refreshSpotifyAccessToken() {
```

Comment:
A same as the comment from above. The updated or new function could use arrow function like this to have less refactoring work later:

const refreshSpotifyAccessToken = async () => {
...
}